glide.lock

```diff
@@ -27,8 +33,12 @@ imports:
   version: fc2b8d3a73c4867e51861bbdd5ae3c1f0869dd6a
   subpackages:
   - pbutil
+- name: github.com/mitchellh/go-homedir
```

Comment:
These aren't our test dependencies; these are getting dragged in with the bumped Consul library version. Thank "vendoring" of library code for that, alas.